src/index.js

```diff
@@ -2,7 +2,10 @@
  * @flow
  */
 
-import React from 'react';
+// import i18next
+import './i18n';
```

Comment:
Please move `i18n.js` to `src/core/i18n/index.js`.

```suggestion
import './core/i18n';
```